src/components/app/details/triggerView/TriggerView.tsx

```diff
@@ -437,17 +437,32 @@ class TriggerView extends Component<TriggerViewProps, TriggerViewState> {
             })
     }
 
-    onClickRollbackMaterial = (cdNodeId) => {
-        ReactGA.event({
-            category: 'Trigger View',
-            action: 'Select Rollback Material Clicked',
-        })
-        getRollbackMaterialList(cdNodeId)
+    onClickRollbackMaterial = (
+        cdNodeId: any,
```

Comment:
should be number or string, we should avoid using any if possible

Reply:
Done. Changed type to `number`.